test/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"cloud.google.com/go/pubsub"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+)
+
+/*
+	Simple program that tests sending a message to a topic and setting up a web server to mimic the sample service api
```

Comment:
This is a nice thing to have around while we're developing it, but I'd like to see us change it for proper integration tests once we're done :)

Reply:
Yeah agree, I actually created an integration test as well. Though this was super handy to develop with, though I will delete at the end.